apps/prairielearn/src/pages/instructorGradingJob/instructorGradingJob.html.ts

```diff
@@ -0,0 +1,210 @@
+                              .grading_job.id}/file/results.json"
+                            >results.json</a
+                          >
+                        </td>
+                        <td>
+                          Contains the PrairieLearn-generated results, which includes the contents
+                          of your
+                          <code>results.json</code> as well as some PrairieLearn metadata.
+                        </td>
+                      </tr>
+                      <tr>
+                        <td>
+                          <a
+                            href="<%= urlPrefix %>/grading_job/<%= grading_job.id %>/file/output.log"
+                            >output.log</a
+                          >
+                        </td>
+                        <td>
+                          Contains the raw output from stdout/stderr for your job. Lines beginning
+                          with
+                          <code>container &gt;</code> are from your container; the rest are
+                          diagnostic logs from PrairieLearn.
+                        </td>
+                      </tr>
+                    </tbody>
+                  </table>
+                </div>
+              `
+            : ''}
+
+          <div class="card mb-4">
+            <div class="card-header bg-primary text-white">Job Output</div>
+            <div class="card-body">
+              ${gradingJobQueryResult.grading_job.s3_bucket &&
+              gradingJobQueryResult.grading_job.s3_root_key
+                ? html`
+                    <script>
+                      $.get('<%= urlPrefix %>/grading_job/<%= grading_job.id %>/file/output.log')
+                        .done(function (data) {
+                          $('#job-output-loading').hide();
+                          $('#job-output').text(data);
+                          $('#job-output').show();
+                        })
+                        .fail(function () {
+                          $('#job-output-loading').hide();
+                          $('#job-output').text('Unable to load grader results');
+                          $('#job-output').show();
+                        });
+                    </script>
+                    <pre
+                      class="bg-dark text-white rounded p-3 mb-0"
+                      id="job-output"
+                      style="display: none;"
+                    ></pre>
+                    <i
+                      class="fa fa-spinner fa-spin fa-2x"
+                      id="job-output-loading"
+                      style="width: 100%; text-align: center;"
+                    ></i>
+                  `
+                : gradingJobQueryResult.grading_job.output
+                ? html`
+                    <pre class="bg-dark text-white rounded p-3 mb-0" id="job-output">
+                        ${gradingJobQueryResult.grading_job.output}
+                        </pre
+                    >
```

Comment:
The `<pre>` tag is whitespace-sensitive, so you'll need to reformat this to avoid excess whitespace. You'll have to run this suggestion through Prettier first.

```suggestion
                    <pre class="bg-dark text-white rounded p-3 mb-0" id="job-output">${gradingJobQueryResult.grading_job.output}</pre>
```